File: AngbandOS.Web/ClientApp/src/app/dashboard/HubConnections.ts
import { ActiveUserDetails } from "./active-user-details";

export class HubConnections {
    public chatHubConnections: ActiveUserDetails[] | undefined;
    public gameHubConnections: ActiveUserDetails[] | undefined;
    public adminHubConnections: ActiveUserDetails[] | undefined;
    public serviceHubConnections: ActiveUserDetails[] | undefined;
    public spectatorsHubConnections: ActiveUserDetails[] | undefined;
}

File: AngbandOS.Web/Models/HubConnections.cs
namespace AngbandOS.Web.Models
{
    public class HubConnections
    {
        public ActiveUserDetails[] ChatHubConnections { get; set; }
        public ActiveUserDetails[] AdminHubConnections { get; set; }
        public ActiveUserDetails[] GameHubConnections { get; set; }
        public ActiveUserDetails[] SpectatorsHubConnections { get; set; }
        public ActiveUserDetails[] ServiceHubConnections { get; set; }
    }
}
